operators/project.py

```diff
@@ -354,9 +354,15 @@ def on_done(future):
             if isinstance(generated, list):
                 generated = generated[-1]
             if texture is None:
-                texture = bpy.data.images.new(name=str(generated.seeds[0]), width=generated.images[0].shape[1], height=generated.images[0].shape[0])
-            texture.name = str(generated.seeds[0])
-            material.name = str(generated.seeds[0])
+                gen_seed_string = str(generated.seeds[0])
+                prompt_subject = bpy.data.scenes["Scene"].dream_textures_project_prompt.prompt_structure_token_subject
+                name_with_prompt = str(prompt_subject + " " + gen_seed_string)
+                texture = bpy.data.images.new(name=name_with_prompt, width=generated.images[0].shape[1], height=generated.images[0].shape[0])
+            gen_seed_string = str(generated.seeds[0])
+            prompt_subject = bpy.data.scenes["Scene"].dream_textures_project_prompt.prompt_structure_token_subject
+            name_with_prompt = str(prompt_subject + " " + gen_seed_string)
```

Comment:
Seems to be some duplication here, could this section be moved above the `if` so it only has to be written once? And an f-string could be simpler here too.

```python
gen_seed_string = str(generated.seeds[0])
prompt_subject = bpy.data.scenes["Scene"].dream_textures_project_prompt.prompt_structure_token_subject
name_with_prompt = f"{prompt_subject} ({gen_seed_string})"
```